ark/dist/chart/templates/crd/ark.mckinsey.com_queries.yaml

```diff
@@ -21,15 +21,15 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
```

Comment:
merge error, we need type, we remove evals, we remove evalselector, we meed query params, this query spec should be same as main